assets/js/main.js

```diff
@@ -314,24 +312,37 @@ function search() {
             var output = '';
             data.results.forEach(function (post, index) {
                 var tooltipDescription = '';
-                var boldedDescription = '';
+                var highlightedTitle = '';
+                var highlightedDescription = '';
                 if (post.meta_description && post.meta_description.raw) {
                     tooltipDescription = post.meta_description.raw;
-                    boldedDescription = post.meta_description.raw.replaceAll(
+                    highlightedTitle = post.title.raw.replaceAll(
                         searchValue,
-                        `<b>${searchValue}</b>`
+                        `<mark>${searchValue}</mark>`
                     );
+                    if (post.meta_description.snippet) {
+                        highlightedDescription = post.meta_description.snippet
+                            .replaceAll(`<em>`, `<mark class="text-xl">`)
+                            .replaceAll(`</em>`, `</mark>`)
```

Comment:
I think using the same size font would be alright here (making it larger might decrease readability slightly). Also, would be best to override the `em` styles to highlight, rather than change `em` tag to `mark`.